### PR TITLE
RT-5.2 aggregate test : Adding type and clearing aggregate link

### PR DESCRIPTION
**Pull request by avaneesh90**
Hi ,

I have added the changes for below github issues in this pull request: 
https://github.com/openconfig/featureprofiles/issues/239
https://github.com/openconfig/featureprofiles/issues/200

Thanks,
Avaneesh

### CODE REVIEW

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 1]
```diff
@@ -217,6 +229,15 @@ func (tc *testCase) configureDUT(t *testing.T) {
...
 
+	//Unset openconfig-lacp:lacp config "Lacp_LacpActivityType_UNSET"
+	if tc.lagType == lagTypeSTATIC {
+		lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
+		lacp.LacpMode = telemetry.Lacp_LacpActivityType_UNSET
+		lacpPath := d.Lacp().Interface(tc.aggID)
+		fptest.LogYgot(t, "LACP", lacpPath, lacp)
+		lacpPath.Replace(t, lacp)
+	}
+
```
Comment (liulk) [inline feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go:240]: Can we merge the two cases so the difference is more obvious?

```
lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
if tc.lagType == lagTypeLACP {
  lacp.LacpMode = telemetry.Lacp_LacpActivityType_ACTIVE
} else {
  lacp.LacpMode = telemetry.Lacp_LacpActivityType_UNSET
}
lacpPath := d.Lacp().Interface(tc.aggID)
fptest.LogYgot(t, "LACP", lacpPath, lacp)
lacpPath.Replace(t, lacp)
```

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 2]
```diff
@@ -182,6 +182,12 @@ func (tc *testCase) setupAggregateAtomically(t *testing.T) {
 	for _, port := range tc.dutPorts[1:] {
 		i := d.GetOrCreateInterface(port.Name())
 		i.GetOrCreateEthernet().AggregateId = ygot.String(tc.aggID)
+		//Adding type configuration "ethernetCsmacd"
```
[the remark this answers is not in the document]
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Done and latest push has the changes

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 3]
```diff
@@ -195,6 +201,10 @@ func (tc *testCase) clearAggregateMembers(t *testing.T) {
 	}
 }
 
+//Deleting aggregate min link configuration from dut
+func (tc *testCase) clearAggregateMinlink(t *testing.T) {
+	tc.dut.Config().Interface(tc.aggID).Aggregation().MinLinks().Delete(t)
+}
```
[the remark this answers is not in the document]
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Done and latest push has blank line

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 4]
```diff
@@ -204,6 +214,8 @@ func (tc *testCase) configureDUT(t *testing.T) {
 	d := tc.dut.Config()
 
 	if *deviations.AggregateAtomicUpdate {
+		//Deleting aggregate min link configuration from dut
```
[the remark this answers is not in the document]
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Done and latest push has the changes.

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 5]
```diff
@@ -195,6 +201,10 @@ func (tc *testCase) clearAggregateMembers(t *testing.T) {
 	}
 }
 
+//Deleting aggregate min link configuration from dut
```
[the remark this answers is not in the document]
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Done and latest push has merge these 2 function clearAggregateMembers and clearAggregateMinlink in clearAggregate

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go (hunk 1 above)
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go:229]: Done and latest push has merge LACP and STATIC

**Review by liulk — APPROVED** [review]

**Review by liulk — CHANGES_REQUESTED** [review]
Please fix the trailing whitespace.

**Review by liulk — APPROVED** [review]

File: feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go [hunk 6]
```diff
@@ -204,18 +213,19 @@ func (tc *testCase) configureDUT(t *testing.T) {
...
 
+	lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
 	if tc.lagType == lagTypeLACP {
-		lacp := &telemetry.Lacp_Interface{Name: ygot.String(tc.aggID)}
 		lacp.LacpMode = telemetry.Lacp_LacpActivityType_ACTIVE
-
-		lacpPath := d.Lacp().Interface(tc.aggID)
-		fptest.LogYgot(t, "LACP", lacpPath, lacp)
-		lacpPath.Replace(t, lacp)
+	} else {
```
[the remark this answers is not in the document]
Comment (avaneesh90) [reply feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go]: Done and latest push has the changes

**Pull request merged**